x-pack/test_serverless/functional/test_suites/observability/dataset_quality/home.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const PageObjects = getPageObjects([
+    'datasetQuality',
+    'observabilityLogsExplorer',
+    'svlCommonNavigation',
+    'svlCommonPage',
+  ]);
+
+  const testSubjects = getService('testSubjects');
+
+  describe('Dataset quality home', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.login();
+    });
+
+    after(async () => {
+      await PageObjects.svlCommonPage.forceLogout();
+    });
```

Comment:
It is not obvious, but `PageObjects.svlCommonPage.login()` is using basic authentication for login and it is not the way customers access the Kibana they use SAML and each project defines set of roles.
Recently we migrated common e2e tests to use SAML in #176221 and I believe it makes sense to use SAML roles in your tests. WDYT?

Reply:
Changed the implementation to use `loginWithRole`.